staging/src/k8s.io/client-go/tools/events/event_broadcaster.go

```diff
@@ -271,9 +271,9 @@ func getKey(event *v1beta1.Event) eventKey {
 	return key
 }
 
-// startEventWatcher starts sending events received from this EventBroadcaster to the given event handler function.
+// StartEventWatcher starts sending events received from this EventBroadcaster to the given event handler function.
```

Comment:
Please move all the changes to events implementation to a separate PR (also see my comment below).

This one should touch only scheduler code and tests related to it.

Reply:
Once this PR will pass tests, friendly ping for this comment.